### ISSUE TITLE
runs into `NameError`

### ISSUE THREAD

**Issue by kjappelbaum** [texworld/betterbib#273]
Hi texworld team, 

with betterbib 7.0.2 and the following bibfile the command `betterbib format bib.bib --sort-fields --align-values --journal-names short --abbrev-first-names` runs into the following error 

```
Traceback (most recent call last):
  File "/Users/kevinmaikjablonka/miniconda3/envs/gemini/bin/betterbib", line 8, in <module>
    sys.exit(main())
  File "<string>", line 3, in wrapper
  File "<string>", line 1485, in _yjVaS
  File "<string>", line 3, in wrapper
  File "<string>", line 928, in _AMK4K
  File "<string>", line 3, in wrapper
  File "<string>", line 932, in _yLjlZ
  File "<string>", line 3, in wrapper
  File "<string>", line 910, in _YH7lw
NameError: name 'bibtex' is not defined
```

bibfile

```
@misc{liang2023holistic,
  title         = {Holistic Evaluation of Language Models},
  author        = {Percy Liang and Rishi Bommasani and Tony Lee and Dimitris Tsipras and Dilara Soylu and Michihiro Yasunaga and Yian Zhang and Deepak Narayanan and Yuhuai Wu and Ananya Kumar and Benjamin Newman and Binhang Yuan and Bobby Yan and Ce Zhang and Christian Cosgrove and Christopher D. Manning and Christopher Ré and Diana Acosta-Navas and Drew A. Hudson and Eric Zelikman and Esin Durmus and Faisal Ladhak and Frieda Rong and Hongyu Ren and Huaxiu Yao and Jue Wang and Keshav Santhanam and Laurel Orr and Lucia Zheng and Mert Yuksekgonul and Mirac Suzgun and Nathan Kim and Neel Guha and Niladri Chatterji and Omar Khattab and Peter Henderson and Qian Huang and Ryan Chi and Sang Michael Xie and Shibani Santurkar and Surya Ganguli and Tatsunori Hashimoto and Thomas Icard and Tianyi Zhang and Vishrav Chaudhary and William Wang and Xuechen Li and Yifan Mai and Yuhui Zhang and Yuta Koreeda},
  year          = 2023,
  eprint        = {2211.09110},
  archiveprefix = {arXiv},
  primaryclass  = {cs.CL}
}
```

I have a hard time debugging this issue. 

Edit: I also run into this for 7.0.1 and 7.0.0

**Comment by nschloe** [comment]
I can reproduce this issue. Will check it out tomorrow.

**Comment by nschloe** [comment]
Fixed in 7.1.0. Running
```
betterbib sync a.bib
```
on your example now correctly gives

```
@article{liang2023holistic,
  url = {http://arxiv.org/abs/2211.09110v2},
  date = {2022-11-16},
  title = {Holistic Evaluation of Language Models},
  author = {Liang, Percy and Bommasani, Rishi and Lee, Tony and Tsipras, Dimitris and Soylu, Dilara and Yasunaga, Michihiro and Zhang, Yian and Narayanan, Deepak and Wu, Yuhuai and Kumar, Ananya and Newman, Benjamin and Yuan, Binhang and Yan, Bobby and Zhang, Ce and Cosgrove, Christian and Manning, Christopher D. and Ré, Christopher and Acosta-Navas, Diana and Hudson, Drew A. and Zelikman, Eric and Durmus, Esin and Ladhak, Faisal and Rong, Frieda and Ren, Hongyu and Yao, Huaxiu and Wang, Jue and Santhanam, Keshav and Orr, Laurel and Zheng, Lucia and Yuksekgonul, Mert and Suzgun, Mirac and Kim, Nathan and Guha, Neel and Chatterji, Niladri and Khattab, Omar and Henderson, Peter and Huang, Qian and Chi, Ryan and Xie, Sang Michael and Santurkar, Shibani and Ganguli, Surya and Hashimoto, Tatsunori and Icard, Thomas and Zhang, Tianyi and Chaudhary, Vishrav and Wang, William and Li, Xuechen and Mai, Yifan and Zhang, Yuhui and Koreeda, Yuta},
  primaryclass = {cs.CL},
  archiveprefix = {arXiv},
  eprint = {2211.09110},
}
```
